src/vs/workbench/parts/search/browser/searchView.ts

```diff
@@ -997,6 +997,7 @@ export class SearchView extends Viewlet implements IViewlet, IPanel {
 		if (resources) {
 			resources.forEach(resource => {
 				let folderPath: string;
+				// #54483 Check with Rob
```

Comment:
@roblourens This PR adds support to open any URI as single folder. It means if there is a file system provider that supports a scheme and a folder from that FSP can be opened like a normal folder.

May I know how this impacts the Search land and what adoptions are needed here? 

I see there are checks made for single folder workbench state and `fspath` is being used, how does the code having this checks is impacted with this?

Reply:
These need to be fixed but search won't work anyway without the search provider API.

Reply:
@roblourens 

> These need to be fixed

Shall I file a separate issue for that?

> search won't work anyway without the search provider API

I see, but when I do search in a workspace with non file scheme folder then I see search progress and it does not stops. In this case (when there is no search provider) is not it nice if we show a message that search is disabled or does not work or et.,?

Reply:
> Shall I file a separate issue for that?

I should probably do it as part of search provider work

> I see, but when I do search in a workspace with non file scheme folder then I see search progress and it does not stops

Yeah we'll have to figure out what to do then. I can show the same message as when you are missing a file system provider.